app/views/welcome/about.html.erb

```diff
@@ -0,0 +1,196 @@
+          </p>
+        </div>
+      </div>
+      <div class="col s12 m12 l4 padding-side-25">
+        <div class="bubble yellow_bg">
+          <div class="bubble-header">3</div>
+          <p class="card-title">
+            We have a great community of event lovers. Share ideas anytime about anything.
+          </p>
+        </div>
+      </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="about aliceblue_bg">
+  <div class="row">
+    <div class="col s12 m12 l12 padding-side-25">
+      <h3 class="teal_text">Our Community.</h3>
+      <p>You can be part of the great work we do just by joining our community of fun-loving people. They love to help... From birthday communities to school clubs, you would definitely find a group you fit into or just create one!</p>
+      <div class="community-row">
+        <%= image_tag 'community.gif' %> 
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="about">
+  <div class="row">
+    <div class="col s12 m12 l12 padding-side-25">
+      <h3 class="teal_text">Meet the Team.</h3>
+      <p>We are a group of fun-loving people who love events! Our team of world-class developers is ready to help you bring your passions to life. Oh, and we are not just developers, we love events too! We have a lot of experience in planning events and know exactly what you want.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col s12 m6 l4">
+      <div class="profile-card">
+        <div class="card-image">
+          <%= image_tag 'team_profile_1.png' %>
+          <div class="feature-details">
+            <div class="feature-title">
+              Michael Ogala
+            </div>
+            <div class="feature-content">
+              <p>Developer</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col s12 m6 l4">
+      <div class="profile-card">
+        <div class="card-image">
+          <%= image_tag 'team_profile_2.jpg' %>
+          <div class="feature-details">
+            <div class="feature-title">
+              Chineze Nwosu
+            </div>
+            <div class="feature-content">
+              <p>Developer</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col s12 m6 l4">
+      <div class="profile-card">
+        <div class="card-image">
+          <%= image_tag 'team_profile_3.jpg' %>
+          <div class="feature-details">
+            <div class="feature-title">
+              Morolake Orekoya
+            </div>
+            <div class="feature-content">
+              <p>Developer</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col s12 m6 l4">
+      <div class="profile-card">
+        <div class="card-image">
+          <%= image_tag 'team_profile_4.jpg' %>
+          <div class="feature-details">
+            <div class="feature-title">
+              Michael Egwali
```

Comment:
CRITICAL: Love the layout/concept but don't approve of my picture being usedon the site. :(